### PR TITLE
Add Link CV

### PR DESCRIPTION
**Pull request by robert-z**

### CODE REVIEW

File: staff/roberto-perez/cv/index.html [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+        </section>
+    </aside>
+    <main>
+        <section>
+            <header>
+                <h2>Experiencia</h2>
+            </header>
+            <ul>
+                <li>
+                    [<time>1998</time>] Ocarina of Time
```
Comment (quinwacca) [inline staff/roberto-perez/cv/index.html]: Why [ ] ?

File: staff/roberto-perez/cv/index.html [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+                <h2>Formación complementaria</h2>
+            </header>
+            <ul>
+                <li>
+                    [<time>1986</time>] Workshop afilador
+                    Herrería de Hyrule
+                    Hyrule
+                </li>
+                <li>
+                    [<time>1985</time>] Curso avanzado de cetrería
```
Comment (quinwacca) [inline staff/roberto-perez/cv/index.html]: 🦅

File: staff/roberto-perez/cv/index.html [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+            </header>
+            <ul>
+                <li>
+                    <address>Hyrule</address>
+                </li>
+                <li>
+                    669 696 696
+                </li>
+                <li>
+                    link@gmail.com
```
Comment (quinwacca) [inline staff/roberto-perez/cv/index.html]: You can hyperlink mail and phone number to make easier any interaction with your cv.
Try 
```javascript
<a href="tel: ...">... </a>
<a href="mailto: ... @ ... .com"> ... @ ... .com</a>
```

Comment (robert-z) [reply staff/roberto-perez/cv/index.html]: Done!

**Comment by gcelma** [comment]
I would delete the aside tag!

**Pull request merged**